plugins/source-terasology-engine/gatsby-node.js

```diff
@@ -0,0 +1,103 @@
+const { graphql } = require("@octokit/graphql");
+const { DateTime } = require("luxon");
+const fs = require("fs");
+
+const PLUGIN_NAME = "source-terasology-engine";
+
+const query = `
+query Engine {
+  organization(login: "MovingBlocks") {
+    repository(name: "Terasology") {
+      issues(first: 10, filterBy: {labels: "Good First Issue", states: OPEN}, orderBy: {field: UPDATED_AT, direction: DESC}) {
+        nodes {
+          id
+          title
+          author {
+            login
+          }
+          labels(first: 10) {
+            nodes {
+              name
+            }
+          }
+          updatedAt
+          url
+        }
```

Comment:
This is (naturally) the same structure as we use for fetching issues for modules. We could use [GraphQL Fragments](https://graphql.org/learn/queries/#fragments) to make sure they always have the same structure.

Which makes me wonder whether this should actually be a separate plugin, or whether we just merge it with the other one for simplicity (and easier sharing of code and query fragments....)

Reply:
let's consider doing that in a follow-up PR :+1: